changelog/_unreleased/2025-04-28-inclue-seourls-for-suggestsearch-and-search-entities.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Inclue seoUrls for suggestSearch and search entities
```

Comment:
```suggestion
title: Include seoUrls for suggestSearch and search entities
```

Reply:
IMO the changelogs should be something like "Allow enriching seoUrls for search results's extensions".

Also please update PR's title and commit message in the same manner